### DISCUSSION TITLE
How to modify the starting and ending points of an edge to be at the center of a portion of a custom node？

### DISCUSSION THREAD

**Discussion by Lzz1027** [bcakmakoglu/vue-flow#1239] [Q&A]
Hi! This is my current graph:
![image](https://github.com/bcakmakoglu/vue-flow/assets/55450873/3da74ccf-7d48-4087-91b8-d3f7992b02ab)
All the edge are started from the bottom and ended at the top, how can i change it?

**Comment by bcakmakoglu** [comment]
Use CSS and place your handle in the center of your node ^^
That's all there is to it.

**Reply by Lzz1027** [reply]
Thanks! I will try! ^_^
